src/privileged/FirefoxMonitor.jsm

```diff
@@ -35,15 +35,15 @@ this.FirefoxMonitor = {
   // the list of breached sites.
   breachListURL: null,
   kBreachListURLPref: "extensions.fxmonitor.breachListURL",
-  kDefaultBreachListURL: "https://haveibeenpwned.com/api/v2/breaches",
+  kDefaultBreachListURL: "https://monitor.firefox.com/hibp/breaches",
```

Comment:
Should we change this to Heroku dev server for testing until it lands in prod?
Otherwise this will 404 (unless we're expecting to quickly push the blurts-server stuff quickly into production).